Documentation/migration-issues.md

```diff
@@ -1,3 +1,21 @@
+## Migration from project.json/xproj to csproj
+
+* For Windows:
+   * Install Visual Studio 2017 RC with the .NET Core workload
+   * Launch Visual Studio 2017 RC
+   * Open your sln or project containing global.json or project.json/xproj
+   * Build 
+   * Run
+
+* For Non-Windows:
+   * Install .NET Core 1.1 SDK
+   * Open the command prompt
+   * Navigate to your project directory containing global.json or project.json/xproj
```

Comment:
If you have a global.json, it would be good to mentioned having to update or remove the SDK version from it, in case you have it there and it points to an older version of the CLI.